gemma-cli/src/main/java/ubic/gemma/core/apps/RNASeqDataAddCli.java

```diff
@@ -53,12 +52,13 @@ public class RNASeqDataAddCli extends ExpressionExperimentManipulatingCLI {
     private static final String MULTIQC_METADATA_FILE_OPT = "multiqc";
     private boolean allowMissingSamples = false;
     private String countFile = null;
-    private boolean isPairedReads = false;
```

Comment:
I found this bug while writing tests: the CLI assumes that data is unpaired if the pairedness is not specified.

Reply:
I'll backport this in the 1.29 and 1.30.

Reply:
Looks like this might be intended looking at the documentation. I don't think it should be backported since it is a breaking change.

Reply:
We need that change because otherwise pairedness can never be null when backfilling.